atomicapp/applogging.py

```diff
@@ -142,11 +149,26 @@ def setup_logging(verbose=None, quiet=None, logtype=None):
 
             # configure logger for color printing to stdout
             handler = logging.StreamHandler(stream=sys.stdout)
-            formatter = colorizeOutputFormatter('%(asctime)s - [%(levelname)s] - %(filename)s - %(message)s')
+            formatter = colorizeOutputFormatter('%(asctime)s - [%(levelname)s] - %(message)s')
             handler.setFormatter(formatter)
-            logger.addHandler(handler)
-            logger.setLevel(logging_level)
+            logger_raw.addHandler(handler)
+            logger_raw.setLevel(logging_level)
             return
 
         # If we made it here then there is an error
         raise Exception("Invalid logging output type: {}".format(logtype))
+
+    @staticmethod
+    def global_logger(filename):
+        """
+        This function returns a logging instance which will output logging event information
+        along with what the LoggerAdapter tells it to output
```

Comment:
paraphrase, "along with LoggerAdapter output" instead of "along with what the LoggerAdapter tells it to output"